ndc.messages.json

```diff
@@ -340,7 +340,7 @@
             "messageClass": "4",
             "messageSubclass": ""
         },
-        "fields": "messageClass,responseFlag,FS,luno,FS,timeVariantNumber,FS,nextState,FS,type1Notes,type2Notes,type3Notes,type4Notes,FS,sernum,function,screen,screenUpdate,screenMessage,FS,coordination,cardReturn,printer,printerData"
+        "fields": "messageClass,responseFlag,FS,luno,FS,timeVariantNumber,FS,nextState,FS,type1Notes,type2Notes,type3Notes,type4Notes,FS,sernum,function,screen,screenUpdate,screenMessage,FS,coordination,cardReturn,printer,printerData,GS,printer1,printerData1"

```

Comment:
This is not a good way to do it. In case there is only one thing to print, GS should not be sent. We cannot be sure if this will work on various versions of NDC. We can send array of `{printer, printData}` objects and join them with GS. Initially printer and printerData were separated for the purpose of easily running automated tests. So this should also be considered.